CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 (Add entries here in the order of: server, console, cli, docs, others)
 
 - console: update sidebar icons for different action and trigger types
+- server: add logs for action handlers
```

Comment:
When you fix the conflict in this file, could you also move the server entry to top.